### ISSUE TITLE
Devices Are Not Being Discovered

### ISSUE THREAD

**Issue by electrolam** [bwssytems/ha-bridge#160]
Hi - I just got the HA-Bridge running and configured, by my echo won't discover the devices.

Here is what I have:
Java running on a Synology DS214se NAS (with latest DSM install).
ASUS RT-AC87R router
Vera Lite (z-wave devices), Sonos, Harmony

I use this to fire up the HA-Bridge:
nohup java -jar ha-bridge-3.1.0.jar -Dupnp.config.address=[Synology NAS IP] &

The HA-Bridge web interface finds my Vera, finds my devices, and everything tests correctly. But whatever I try, the Echo will not discover anything. I've scoured these forums until my eyes are red - and I've tried a bunch of things, even back to going basic and rebooting everything. Nothing seems to work. Does anyone maybe have something I could try?

Last two lines of my nohup file:
2016-09-06 18:11:18,919 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener starting....
2016-09-06 18:11:18,922 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener running and ready....

**Comment by electrolam** [comment]
Attached is my log file
[2016-09-6-habridge_log.txt](https://github.com/bwssytems/ha-bridge/files/458251/2016-09-6-habridge_log.txt)
. I copied what you told a previous user to do and left all the logs at INFO and did the com.bwssystems.HABridge.hue.HueMulator at DEBUG and also turned on the upnp trace. It looks like my echo is reaching out to request the light list...

**Comment by bwssytems** [comment]
So, it looks like all is in line. Did you generate devices for the list? Just by finding your vera it does not auto add them, you have to hit the green button to the side.

**Comment by electrolam** [comment]
Yes, I generated devices. I actually tried something new that worked. I changed the upnp port to 50088. I read somewhere that Synology reserves port 50000. That still didn't work but I started seeing upnp calls from my sonos devices in the log. As a last ditch effort, I powered down all of my sonos devices and tried again. It worked! I'm now running two bridges with 38 devices. 

Thanks to all of your hard work in developing this. This is amazing!
